plugins/a11ychecker.md

```diff
@@ -161,91 +295,212 @@ If `a11ychecker_allow_decorative_images` is set to `false`, the Accessibility Ch
+  }
+});
+```
+
+#### Example: Using `a11ychecker_filter_issue` filter all Accessibility Checker table rules
+
+The callback function in the following example will return `false` if the issue `element` value is `'table'`, filtering all table-related issues from the Accessibility Checker report.
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_filter_issue: function(issue) {
+    return issue.element.nodeName.toLowerCase() !== 'table';
+  }
 });
 ```
 
 ### `a11ychecker_html_version`
 
 This configuration option sets the HTML version to use when checking issues.
 
-For example, setting the version to HTML 4 will trigger the rule "Complex tables should have summaries" as summary is a valid attribute and is required for tables, however in HTML 5 the attribute is deprecated, so the rule won't be triggered.
+For example: Setting the version to HTML 4 will trigger the rule "Complex tables should have summaries", where `summary` is a valid attribute in HTML 4 but not HTML 5.
 
 **Type:** `String`
 
 **Default value:** `html4`
 
 **Possible Values:** `html4`, `html5`
 
-#### Example: a11ychecker_html_version
+#### Example: Using `a11ychecker_html_version`
 
 ```js
 tinymce.init({
-    selector: 'textarea',
-    plugins: 'a11ychecker',
-    toolbar: 'a11ycheck',
-    a11ychecker_html_version: 'html5'
+  selector: 'textarea',
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_html_version: 'html5'
+});
+```
+
+### `a11ychecker_ignored_rules`
+
+The `a11ychecker_ignored_rules` option prevents specific Accessibility Checker rules being checked. This feature allows developers to skip rules that flag known issues in content until the issue is fixed.
+
+**Type:** `String`
+
+**Default Value:** `''`
+
+**Possible values:** `'D1'`, `'D2'`, `'D3'`, `'D4O'`, `'D4U'`, `'D5A'`, `'D5B'`, `'D5C'`, `'H93'`, `'I1'`, `'I2'`, `'T1'`, `'T2'`, `'T3'`, `'T4A'`, `'T4B'`, `'T4C'`
+
+#### Example: Using `a11ychecker_ignored_rules`
+
+This examples shows how to ignore the following checks (rules):
+
+* D2 - Sequential headings
+* I2 - Alt text filename
+* T4B - Table headers
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_ignored_rules: 'D2, I2, T4B'
+});
+```
+
+### `a11ychecker_issue_url_callback`
+
+The `a11ychecker_issue_url_callback` option is used to change the target URL for the "Click for more info" button (![help icon - a question mark inside a circle]({{site.baseurl}}/images/icons/help.svg)) in the Accessibility Checker dialog. By default, the "more info" links will point to `{{site.url}}/docs/plugins/a11ychecker/#<ruleId>`, such as `{{site.url}}/docs/plugins/a11ychecker/#D1`. This option can be used to set the target URL to a page or pages outside {{site.url}}/docs/.
+
+**Type:** `Function`
+
+**Default Value:** `undefined`
```

Comment:
Should we tell a small lie here and say that the default value is a function that returns `{{site.url}}/docs/plugins/a11ychecker/#${ruleId}` (as even if it might not be done through a default parameter, that's the default behaviour when you don't set it)? Or do you think the example use below provides enough of an example for the style of function to write here

Reply:
> By default, the "more info" links will point to `{{site.url}}/docs/plugins/a11ychecker/#<ruleId>`, such as `{{site.url}}/docs/plugins/a11ychecker/#D1`

Already stated in the description to reduce confusion a bit (I hope)